std_msgs/Header header
geometry_msgs/Point[] cluster_center
geometry_msgs/Point[] cluster_min
geometry_msgs/Point[] cluster_max